elastic/kibana
# [Inventory][ECO] Entities Group By View

# Summary

This PR introduces the API and Page for doing grouped views for the Inventory Page. Alongside the plain list view, the page now by default shows a grouped view of entities. In this PR, the only current supported grouping is by Entity Type.

https://github.com/user-attachments/assets/a07db592-d6c6-4ec1-a00b-bb469908aa6a

Tests TBA

## How to test

- Navigate to the new Inventory Page
- By default, the page should load into a grouped view (Type)
- The page should show all entities currently grouped by their type.
- If a group has enough entities, pagination navigation should only apply to the list within the group.
- The plain list view should function same as before.
- Using the search/filter bar should function the same with grouped and list view.

Closes #194740
